pandas/io/sas/sasreader.py

```diff
@@ -48,7 +48,7 @@ def read_sas(filepath_or_buffer, format=None, index=None, encoding=None,
             else:
                 raise ValueError("unable to infer format of SAS file")
         except ValueError:
-            pass
+            raise
```

Comment:
https://github.com/pandas-dev/pandas/issues/24548#issuecomment-450906323

Reply:
yeah you can see if this is removable

Reply:
Sure